torch/csrc/WindowsTorchApiMacro.h

```diff
@@ -5,3 +5,9 @@
 // There's no difference between aten, torch and caffe2 libs any more
 // TODO: clean up the naming for consistency
 #define TORCH_API CAFFE2_API
+
+#ifdef _WIN32
+#define TORCH_PYTHON_API
+#else
+#define TORCH_PYTHON_API TORCH_API
+#endif
```

Comment:
Wouldn't that definition work for both platforms?
```suggestion
#define TORCH_PYTHON_API C10_EXPORT
```